3rdparty/python/requirements.txt

```diff
@@ -14,9 +14,9 @@ humbug==0.2.7
 
 ijson==3.1.4
 packaging==21.3
-pex==2.1.66
+pex==2.1.67
 psutil==5.9.0
-pytest>=6.2.4,<8  # This should be compatible with pytest.py, although it can be looser so that we don't over-constrain pantsbuild.pants.testutil 
+pytest>=6.2.4,<8  # This should be compatible with pytest.py, although it can be looser so that we don't over-constrain pantsbuild.pants.testutil
```

Comment:
This was confusingly unclean. It's trailing white-space; so I was not too concerned.